src/core_modules/capture-ui/FlatList/flatList.types.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+import * as React from 'react';
+
+export type Props = {
+    list: any[],
+    renderItem: (item: any) => React.Element<any>,
+    className?: string,
+    dataTest?: string,
+};
```

Comment:
Can we try to be exact in this annotation over here or you reckon it is impossible?

Reply:
good point! I made it `any` so that it can fit all kinds of object

Reply:
Yes that's cool I think, but I was referring more to the whole `Props` annotation. 

Maybe would be easier if I post this https://flow.org/en/docs/types/objects/#toc-exact-object-types for reference.

Basically when you annotate exactly you will get an flow error if someone is trying to pass another property in a component. As we have it now I could pass any property I might want in the `FlatList` and there wont be an error warning me I am doing something wrong. In my eyes it is quite useful to have these errors in place so that we developers can communicate with each other of the appropriate way a component is meant to be used.